### PR TITLE
Fix for permissions and applications not creating

### LINKED ISSUE #67: Permissions are not enforced
1. Create a new user with `admin` flag set to `nil`
2. Log in
3. Try to manipulate Faculties or Organizations

Expected result: I don't have any access to those views, nor via menus nor by directly going to the URL

Actual result: I can manipulate Faculties and Organizations as a normal user.

### LINKED ISSUE #73: Can't create new Application
In ApplicationsController#create L36 you tried to use method `organization_params` which does not exist in that controller.

Screenshot_2019-04-12 Finansowanie

### PR DESCRIPTION
**Pull request by TeachMeHow**
Fixed #73 by correcting parameter name
Fixed #67 - user being able to manipulate faculties, by enforcing authorization

### CODE REVIEW

File: app/controllers/faculties_controller.rb [hunk 1]
```diff
@@ -7,30 +7,37 @@ class FacultiesController < ApplicationController
     redirect_to request.referer || root_path
   end
   before_action :set_faculty, only: %i[show edit update destroy]
-
+  after_action :verify_authorized
+  j
```
Comment (Quintasan) [inline app/controllers/faculties_controller.rb]: Remove this

File: app/views/applications/_form.html.haml [hunk 2]
```diff
@@ -105,6 +105,7 @@
               .container.schedule_items_row.w-100.my-4.p-3.border.border-secondary
                 .row
                   .col-md-4
+                    // TODO - whatever is todo???
```
Comment (Quintasan) [inline app/views/applications/_form.html.haml]: If I recall correctly a `Todo` is an item that represents things to be done on some schedule. This model should be called `ScheduleItem` I guess.

File: spec/rails_helper.rb [hunk 3]
```diff
@@ -1,25 +1,18 @@
...
 require 'pundit/rspec'
 require 'pundit/matchers'
 require 'database_cleaner'
-# frozen_string_literal: true
-
 require 'spec_helper'
-require File.expand_path('../config/environment', __dir__)
-abort("The Rails environment is running in production mode!") if Rails.env.production?
-require 'rspec/rails'
-require 'pundit/rspec'
```
Comment (Quintasan) [inline spec/rails_helper.rb]: Why did you remove all those requires?

File: app/views/applications/_form.html.haml (hunk 2 above)
Comment (TeachMeHow) [reply app/views/applications/_form.html.haml]: I'll rename it then. "todo" is very non-descriptive

File: spec/rails_helper.rb (hunk 3 above)
Comment (TeachMeHow) [reply spec/rails_helper.rb]: Reason being, in the original file they were duplicated, as can be seen here.

File: app/views/applications/_form.html.haml (hunk 2 above)
Comment (Quintasan) [reply app/views/applications/_form.html.haml]: Okay, resolve this comment once done. Remember you have to rename:
1. Model.
2. Table in the database.
3. All occurennces of `:todo` in relations in other models and tables.

Comment (Quintasan) [reply app/views/applications/_form.html.haml]: @TeachMeHow Did you finish that one?

**Pull request merged**